package.json

```diff
@@ -45,14 +45,15 @@
     "babel-core": "^6.26.0",
     "babel-jest": "^21.2.0",
     "babel-loader": "^7.1.2",
-    "babel-plugin-flow-react-proptypes": "^10.0.0",
     "babel-plugin-module-resolver": "^3.0.0",
     "babel-plugin-transform-class-properties": "^6.24.1",
     "babel-plugin-transform-imports": "^1.4.1",
     "babel-plugin-transform-object-rest-spread": "^6.26.0",
     "babel-preset-env": "^1.6.0",
     "babel-preset-react": "^6.24.1",
     "babel-preset-react-optimize": "^1.0.1",
+    "babel-plugin-flow-react-proptypes": "^10.0.0",
+    "babel-plugin-transform-flow-strip-types": "^6.22.0",
```

Comment:
Sorry to be totally pedantic, but these need to be in alphabetical order or npm will reorder them next time we install something, which adds unrelated noise to future PRs.

In my defence, it's really npm that's being pedantic here 😄

Reply:
haha! https://www.youtube.com/watch?v=BI5IA8assfk

😂

Reply:
Oh gawd, the worst part about pasting that into youtube is I'm actually seat dancing to it right now. Should be all good.